ci build failure again in FlowRunnerTest

see https://travis-ci.org/azkaban/azkaban/builds/242197329?utm_source=email&utm_medium=notification

azkaban.execapp.FlowRunnerTest > exec1Disabled FAILED
    java.lang.AssertionError: Flow didn't reach expected status
        at org.junit.Assert.fail(Assert.java:88)
        at azkaban.execapp.FlowRunnerTestBase.waitFlowRunner(FlowRunnerTestBase.java:52)
        at azkaban.execapp.FlowRunnerTestBase.assertThreadShutDown(FlowRunnerTestBase.java:29)
        at azkaban.execapp.FlowRunnerTest.exec1Disabled(FlowRunnerTest.java:144)
74 tests completed, 1 failed, 22 skipped
:azkaban-exec-server:test FAILED

--- a/azkaban-exec-server/src/test/java/azkaban/execapp/FlowRunnerTestBase.java
+++ b/azkaban-exec-server/src/test/java/azkaban/execapp/FlowRunnerTestBase.java
@@ -38,7 +38,7 @@ public class FlowRunnerTestBase {
   }
 
   public void waitFlowRunner(final Function<FlowRunner, Boolean> statusCheck) {
-    for (int i = 0; i < 100; i++) {
+    for (int i = 0; i < 1000; i++) {
       if (statusCheck.apply(this.runner)) {
         return;
       }
@@ -54,7 +54,7 @@ public class FlowRunnerTestBase {
 
   public void waitJobStatuses(final Function<Status, Boolean> statusCheck,
       final String... jobs) {
-    for (int i = 0; i < 100; i++) {
+    for (int i = 0; i < 1000; i++) {
       if (checkJobStatuses(statusCheck, jobs)) {
         return;
       }
@@ -74,7 +74,7 @@ public class FlowRunnerTestBase {
 
   protected void waitEventFired(final String nestedId, final Status status)
       throws InterruptedException {
-    for (int i = 0; i < 100; i++) {
+    for (int i = 0; i < 1000; i++) {
       for (final Event event : this.eventCollector.getEventList()) {
         if (event.getData().getStatus() == status && event.getData().getNestedId()
             .equals(nestedId)) {

